README.md

```diff
@@ -159,13 +159,13 @@ Latest official release:
 <dependency>
   <groupId>nl.jiankai</groupId>
     <artifactId>spotify-web-api-wrapper</artifactId>
-    <version>1.5.7</version>
+    <version>1.5.8</version>
```

Comment:
Since this PR introduces a breaking change to the API I'm unsure yet whether it should be part of a major version release. It definitely won't be a patch version release.

Reply:
When this PR gets approved I might merge it to a next major version release branch and shelve it until the time is right.

Reply:
What do you think, which version should I use? Asking because the same version should be added in the Javadocs of deprecated methods, too.

Reply:
It should be 2.0.0 and should target a 2.0.0 release branch instead of `main`. We can later merge it into master or cherry pick the commits.

Reply:
Thank you. I updated the version numbers, but I think only you can create the necessary release branch, that I can then set as a target (or close this PR and open a new one, as you see fit).